src/schoology-extractor/schoology_extractor/helpers/sync.py

```diff
@@ -85,28 +95,26 @@ def _table_exist(
 
 def _remove_duplicates(
     resource_name: str,
-    db_engine: sqlalchemy.engine.base.Engine,
-    unique_id_column: str
+    db_engine: sqlalchemy.engine.base.Engine
 ):
     with db_engine.connect() as con:
         con.execute(
             f"DELETE from {resource_name} "
             "WHERE rowid not in (select max(rowid) "
             f"FROM {resource_name} "
-            f"GROUP BY {unique_id_column})"
+            f"GROUP BY SourceId)"
```

Comment:
I have never seen a `DELETE` statement with a `GROUP BY` statement before! Nice technique.

Reply:
Google Classroom does this too.  😎

Reply:
Got it from Brad's code 😄